tools/lsp/preview/ui.rs

```diff
@@ -1,28 +1,46 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
+use i_slint_core::{
+    model::{Model, VecModel},
+    SharedString,
+};
 use slint_interpreter::PlatformError;
 
-slint::slint!(
-import { Button, VerticalBox } from "std-widgets.slint";
-
-export component PreviewUi inherits Window {
-    in property<component-factory> preview_area <=> preview_area_container.component-factory;
-    callback design_mode_changed(bool);
-
-    VerticalBox {
-        // Button {
-        //     text: "Design Mode";
-        //     checkable: true;
-        //     clicked => { root.design_mode_changed(self.checked); }
-        // }
-        preview_area_container := ComponentContainer {}
-    }
-}
-);
+slint::include_modules!();
+
+pub fn create_ui(style: String) -> Result<(PreviewUi, String), PlatformError> {
+    let default_style = "fluent-light";
```

Comment:
still need to be addressed: We really need to try to make the preview as close as possible as the app, and the app uses native by default.
I know that we don't ship the preview with Qt, but on mac now we want the cupertino style and this is also a regression because of the dark style

Reply:
It should not use the "qt" style, but the "native" style, right?
The "native" style is an alias to whatever default we use on the platform, see 
https://github.com/slint-ui/slint/blob/589ab56643d29689ca0052e76d2cf91ea7cbce77/internal/compiler/typeloader.rs#L95-L97
and there is also some extra logic in the interpreter for it.

But note that the "native" style is not in the widgets style from file access, so you need to add it explicitly like so:
https://github.com/slint-ui/slint/blob/589ab56643d29689ca0052e76d2cf91ea7cbce77/internal/compiler/typeloader.rs#L106-L107